lib/system/database.rb

```diff
@@ -71,7 +71,7 @@ def ready?
 
         def sql_for_readiness
           if oracle?
-            "SELECT 1 FROM v$pdbs WHERE name COLLATE BINARY_CI = '#{connection_config[:database]}' AND open_mode = 'READ WRITE'"
+            "SELECT 1 FROM v$database WHERE open_mode = 'READ WRITE'"
```

Comment:
This is OK because the connection should be with porta's pdb already.
https://github.com/3scale/porta/blob/db715a8e350c88840fd173849eeb1c09e4a41ab3/lib/system/database.rb#L64

Reply:
I see that `sql_for_readiness` is a public class method. So, is it called from elsewhere? In that case we shouldn't have gone through that line yet 😞

Reply:
It is not used elsewhere. Actually the whole `ConnectionProbe` class is only used by `Database#ready?`

Reply:
Well actually you can `connect` but you cannot use it as the PDB is not ready yet